getting-started.md

```diff
@@ -0,0 +1,64 @@
+# **Getting Started**
+
+> Ok, I have installed Allstar on my account/organization, now what?
+
+By default, Allstar will not take any actions to avoid unwanted noise. To
+quickly enable Allstar on all of your repos:
+
+1. Create a repository named `.allstar`
+1. Create a file named `allstar.yaml` with the contents
+
+       optConfig:
+         optOutStrategy: true
+
+1. Create a file named `branch_protection.yaml` with the contents
```

Comment:
We should mention that we are just starting with this one check and more checks will get added soon.